verifiers/utils/builder.go

```diff
@@ -0,0 +1,22 @@
+package utils
```

Comment:
WDYT about adding a function that compares builder IDs? otherwise you have logic for comparing repeated across tests and in code. It would capture the logic that if `expected` contains the version, then it compares that too.

Reply:
Good idea. I've been thinking about doing it as well, but was not sure. Let me add it.

Reply:
done. PTAL